spring-cloud-task-core/src/main/java/org/springframework/cloud/task/repository/dao/MapTaskExecutionDao.java

```diff
@@ -139,6 +140,17 @@ public long getRunningTaskExecutionCountByTaskName(String taskName) {
 		return count;
 	}
 
+	@Override
+	public long getRunningTaskExecutionCount() {
+		int count = 0;
```

Comment:
int needs to be a long

Reply:
May also consider replacing the for loop with: 
```
count = taskExecutions.entrySet().stream()
	.filter(entry -> entry.getValue().getEndTime() == null)
	.count();
```

Reply:
I agree on the `long`, I personally disagree with the changing a for loop to a stream just for syntactic sugar.  I'm personally a fan of not using streams for this use case (it's a performance hog: https://jaxenter.com/java-performance-tutorial-how-fast-are-the-java-8-streams-118830.html).